lib/ts/recipe/dashboard/api/user/create/passwordlessUser.ts

```diff
@@ -0,0 +1,121 @@
+import { APIInterface, APIOptions } from "../../../types";
+import STError from "../../../../../error";
+import Passwordless from "../../../../passwordless";
+import ThirdPartyPasswordless from "../../../../thirdpartypasswordless";
+import PasswordlessRecipe from "../../../../passwordless/recipe";
+import ThirdPartyPasswordlessRecipe from "../../../../thirdpartypasswordless/recipe";
+import { User } from "../../../../../types";
+import RecipeUserId from "../../../../../recipeUserId";
+import { parsePhoneNumber } from "libphonenumber-js/max";
+
+type Response =
+    | {
+          status: string;
+          createdNewRecipeUser: boolean;
+          user: User;
+          recipeUserId: RecipeUserId;
+      }
+    | {
+          status: "FEATURE_NOT_ENABLED_ERROR";
+      }
+    | {
+          status: "EMAIL_VALIDATION_ERROR";
+          message: string;
+      }
+    | {
+          status: "PHONE_VALIDATION_ERROR";
+          message: string;
+      };
+
+export const createPasswordlessUser = async (
+    _: APIInterface,
+    tenantId: string,
+    options: APIOptions,
+    __: any
+): Promise<Response> => {
+    let passwordlessOrThirdpartyPasswordlessRecipe: PasswordlessRecipe | undefined = undefined;
+
+    try {
+        passwordlessOrThirdpartyPasswordlessRecipe = PasswordlessRecipe.getInstanceOrThrowError();
+    } catch (_) {
+        try {
+            passwordlessOrThirdpartyPasswordlessRecipe = ThirdPartyPasswordlessRecipe.getInstanceOrThrowError()
+                .passwordlessRecipe;
```

Comment:
Using `passwordlessRecipe` property in the `ThirdpartyPasswordlessRecipe` since if the user didn't defined the validator function in the config then the functions are `undefined`  even if they have defined validators those will be passed down to this `passwordlessRecipe` property according to these [line](https://github.com/supertokens/supertokens-node/blob/master/lib/ts/recipe/thirdpartyemailpassword/recipe.ts#L128) of code.